snaps/openvino/README.md

```diff
@@ -92,6 +95,12 @@ after you have saved the model to OpenVINO IR format, and before you have compil
 with `compiled_model = core.compile_model(model, device)`. You should re-run the cell
 that saves or loads the model after patching the xml file.
 
+Additionally, because the app does not have permissions to access a user's home
+directory, a different model path must be used compared to the one used in
+the upstream Intel Jupyter notebook. A good choice for the model path is
+`/home/ubuntu/snaps/openvino/current` as this is accessible both inside and outside
```

Comment:
`/home/ubuntu/snap/...` instead of  `/home/ubuntu/snaps/....`  ?

Reply:
Good catch! I'll push a fix.

Reply:
Fixed.